apply.sh

```diff
@@ -6,18 +6,18 @@ set -e
 # Patch EKS nodes
 # 
 
-if [ -z $1 ]; then
-    echo "Error: cluster variable is not set, I don't know which cluster you want to create"
+if [ -z $1 ] || [ -z $2 ]; then
+    echo "Error: workspace and/or workspaces path variables not set, I don't know which workspace you want to create"
     exit 1
 fi
 
 export WORKSPACE=$1
+export WORKSPACESPATH=$2
```

Comment:
```suggestion
export WORKSPACESPATH=$(realpath $2)
```

`realpath` will produce an absolute path from a relative path if provided but will still preserve an absolute path.